extensions/amp-iframe/1.0/component.js

```diff
@@ -40,8 +41,8 @@ export function Iframe({
 
   const attemptResize = useCallback(() => {
     const iframe = iframeRef.current;
-    let height = Number(dataRef.current.height);
-    let width = Number(dataRef.current.width);
+    let height = Number(dataRef.current?.height);
+    let width = Number(dataRef.current?.width);
     if (!height && !width) {
       console./*OK*/ error(
         'Ignoring resize request because width and height value is invalid'
```

Comment:
Nit: I would refactor this so that it's if `isIntersectingRef.current ` is truthy, it's not called. And if there's not a pending resize, it's not called. etc.